.vscode/vetur/snippets/template-typescript-w-decorator.vue

```diff
@@ -1,13 +0,0 @@
-<template>
```

Comment:
maybe would be good to have snippet for composition API?

Reply:
good point. maybe we need to list recommended extensions in the readme files. this extension seems good already https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-snippets. any idea where to put that list?

Reply:
I put recommended extensions in here https://github.com/preschian/nft-gallery/blob/init-histoire-and-workspaces/CONTRIBUTING.md#getting-started